jekyll/_layouts/classic-docs.html

```diff
@@ -14,10 +14,11 @@
 				<input type="text" autocomplete="off" placeholder="What can we help you find?" name="query" class="form-control" />
 				<ul id="search-results" class="list-unstyled dropdown-menu"></ul>
 			</form>
+			{% if page.page-type == "index" %}<h2>{{ page.title }}</h2>{% endif %}
 			<article>
-				<h1>{{ page.title }}</h1>
+				{% if page.page-type != "index" and page.page-type != "homepage" %}<h1>{{ page.title }}</h1>{% endif %}
 				{{ content }}
-				{% if page.path != "index.html" and page.layout == "classic-docs" or page.layout == "enterprise" %}
+				{% if page.page-type != "index" and page.layout == "classic-docs" or page.layout == "enterprise" %}
```

Comment:
This is intentional. The page doesn't look good without the footer when there is no sidebar menu. Improvements planned for the landing page after this PR.